item.rb

```diff
@@ -1,16 +1,20 @@
 require 'date'
 
 class Item
```

Comment:
- [ ] You have all done a great job with your project so far :clap: There is a method that seems to not be included in your item class. You might have forgotten to add the label/ add_label method in your items class. Kindly include it to meet all the project requirements.